kythe/go/indexer/indexer.go

```diff
@@ -57,6 +58,7 @@ import (
 	"github.com/golang/protobuf/proto"
 	"golang.org/x/tools/go/gcexportdata"
 
+	mpb "kythe.io/kythe/proto/metadata_go_proto"
```

Comment:
This is likely going to require running `go test -mod=mod ./kythe/...` to update the go.mod file.

Reply:
go: finding module for package kythe.io/kythe/proto/metadata_go_proto
kythe/go/indexer/indexer.go:63:2: no matching versions for query "latest"

Reply:
I used Python2 to run generate_protobufs.py. However, even so, subsequently running go test -mod=mod ./kythe/... still results in no modified or added files. I get lengthy output in my terminal with some errors and failures.